Visualizer/src/components/dashboard/TopBar.vue

```diff
@@ -97,7 +89,7 @@ const stats = computed(() => {
     {
       label: "Collisions",
       value: simulation.statistics.totalNumberOfCollisions,
```

Comment:
```suggestion
      label: "Violations",
      value: simulation.statistics.totalNumberOfCollisions,
```

Reply:
rename property as well. Should be called violations.

Reply:
I don't get it. The API property is called "total_number_of_collisions", I name it "totalNumberOfCollisions" and display it under the title "Collisions", but you are telling me it should be called Violations?

Reply:
habs in meinem PR gefixed